cmd/juju/commands/bootstrap.go

```diff
@@ -93,10 +93,16 @@ address.
 
 Private clouds may need to specify their own custom image metadata and
 tools/agent. Use '--metadata-source' whose value is a local directory.
-The value of '--agent-version' will become the default tools version to
-use in all models for this controller. The full binary version is accepted
-(e.g.: 2.0.1-xenial-amd64) but only the numeric version (e.g.: 2.0.1) is
-used. Otherwise, by default, the version used is that of the client.
+
+By default, the Juju version of the bootstrapping client is the version of the
+tools agent that is downloaded and installed for all models on this controller.
+However, a user can specify a different agent version via '--agent-version' 
+option to bootstrap command. Juju will use this version for models' agents 
+as long as the client's version is from the same Juju release series.
+In other words, a 2.0.1 client can bootstrap any 2.0.x agents but cannot
+bootstrap any 1.25.x or 2.1.x agents.
+The agent version can be a simple numeric version, e.g. 2.0.1,
+or a full binary version, e.g. 2.0.1-xenial-amd64. 
```

Comment:
I didn't realise this - does this work? I usually use --bootstrap-series to specify the series, and a constraint for the arch. Does doing this set those values implcitly?

Reply:
This was described on the command before. I think it maybe best to omit this format altogether, especially since we prefer the numeric one anyway :)